Guides/Trim.md

```diff
@@ -48,6 +48,25 @@ func myAlgorithm2<Input>(input: Input) where Input: BidirectionalCollection {
 Swift provides the `BidirectionalCollection` protocol for marking types which support reverse traversal,
 and generic types and algorithms which want to make use of that should add it to their constraints.
 
+### >= 0.3.0
```

Comment:
I added `0.3.0` supposing that this will be released in a next minor version, but I am not sure